### ISSUE TITLE
Wish: Looking at fit parameters

### ISSUE THREAD

**Issue by ajasja** [jungmannlab/picasso#239]
Picasso version: 0.4 (pip)
Python version: 3.8.5
Operating System: Win 10

### Description

I'm looking for the best way to filter the fitted localizations. I would like to click on a fit and see the parameters. Is there an easy a way to do this?

### What I Did

Localized and fitted a tiff file. I'm getting several spurious fits that should be easy to filer away, but I don't have a feeling for which fit parameters are best to filter on.  
![image](https://user-images.githubusercontent.com/19926/166438388-2d0c4bb1-fd97-4d14-97e1-f913dc8dc29b.png)

If I could click on a localization and see the fit parameters that would be great.

**Comment by cleterrier** [comment]
Here it looks like a simple filtering on x and Y coordinates (being close to the edge) would catch all these unwanted localizations?

**Comment by ajasja** [comment]
@cleterrier indeed, that's what I'm doing now, but there are still spurious fits in other frames in the middle of the image.

**Comment by ajasja** [comment]
It seems like the FitMarker (https://github.com/jungmannlab/picasso/blob/ac0064fbc232caab1135ebd3c4cc55f439a1cd32/picasso/gui/localize.py#L158) could be extended with mouse over or click events relatively easily.

**Comment by ajasja** [comment]
@rafalkowalewski1 Thanks! I hope I solved the problem by only looking at spots near the fibers, but I still think it would be cool if you could click on the fit and see the parameters

**Comment by rafalkowalewski1** [comment]
Hello, the localization parameters will be shown in version 0.11.0 and 0.11.0a4 which will be available next week.

Sorry for taking 4.5 years to do this.

Best,
Rafal

**Comment by ajasja** [comment]
Thanks! Still useful for us! :)
